ckanext/ckanext-apply_permissions_for_service/ckanext/apply_permissions_for_service/migration/apply_permissions_for_service/versions/e2fdr852000b_add_intermediate_organization.py

```diff
@@ -0,0 +1,28 @@
+"""Add application file
+
+Revision ID: e2fdr852000b
+Revises:
+Create Date: 2022-04-08 11:45:38.997096
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'e2fdr852000b'
+down_revision = None
```

Comment:
```suggestion
down_revision = 'e2fce751960a'
```

Migrations need to have an order in which they are applied, if both are none the migration command will fail.